site/content/entry/citra-progress-report-2017-july.md

```diff
@@ -0,0 +1,61 @@
+<br />
+<!--
+title_id = 0004000000174600
+commit_hash = a40e60c4bc7ab9c559ad6e3f1769b1128d68c8d5
+-->
+*How jagged*
+</p>
+
+Fortunately (and to much rejoice) wwylele stepped into the ring to wrestle with this issue. They prepared a [test program](https://github.com/wwylele/ctrhwtest/tree/master/texture-copy-test) to help gain an understanding of how hardware handles the TextureCopy operation in comparison to Citra.
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="50%" width="50%" alt="Pokemon Super Mystery Dungeon During Deoxy's and Rayquaza's Face Off (IN S P A C E)" src="/images/entry/citra-progress-report-2017-july/texturecopy-after.png" />
+<br />
+<!--
+title_id = 0004000000174600
+commit_hash = 57438cf51f3bdae71a694d2ebe96ad563968781c
+-->
+Deoxys is having a bit of a hard time, no?
+</p>
+
+
+## [Citra-QT: UI Themes](https://github.com/citra-emu/citra/pull/2804) by [Kloen](https://github.com/kloen)
+
+[Kloen](https://github.com/kloen) has put the time and work into Citra's Qt frontend to make it themable! Now users can enjoy a dark mode and other custom coloring schemes!
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="75%" width="75%" alt="Comparison of Dark Theme and Light Theme" src="/images/entry/citra-progress-report-2017-july/theme-comparison.png" />
+<br />
+<!--
+commit_hash = 081304eca57913cd2cc67f35b9b49a87a6f0b2c2
+-->
+CHOOSE YOUR CHARACTER
+</p>
+
+## [Load Shared Font From System Archive](https://github.com/citra-emu/citra/pull/2784) by [wwylele](https://github.com/wwylele)
+
+**This does not elimate the need for dumping a shared font from a legitimate system.**
+
+This in itself isn't an extremely visible or perceivable change from a user perspective, but it is something that helps us take another stride towards accurately recreating the way the 3DS actually operates. All system data is now uniformly stored in the system archive now that the system font can be included with it. Although, Citra does still fall back to the now deprecated `shared_font.bin` file that was being dumped before for compatibility's sake. `3dsutil` has been updated to be able to dump everything as a system archive so that Citra may be able to work with it in a way that's more accurate to the actual hardware!
+
+## To Contributors
+
+Thank you to [everyone who's contributed](https://github.com/citra-emu/citra/graphs/contributors?from=2017-07-10&to=2017-08-16&type=c) for months before, beyond, and during July for all of the work that you've put into Citra. Without all of you the project would not have shown the progress that it has in the past year and with you all I don't think it's showing any sign of slowing down!
```

Comment:
"Without all of you, the project would not have shown the progress that it has in the past year, and with all those who have contributed, I don't think it's showing any sign of slowing down!"